src/main/java/org/zone/commands/structure/ZoneCommands.java

```diff
@@ -77,6 +78,7 @@
     ZoneFlagGreetingsViewCommand ZONE_FLAG_GREETINGS_VIEW_COMMAND = new ZoneFlagGreetingsViewCommand();
     ZoneFlagInteractItemframesViewCommand ZONE_FLAG_INTERACT_ITEMFRAMES_VIEW_COMMAND = new ZoneFlagInteractItemframesViewCommand();
 
+    EditBoundsStartCommand EDIT_BOUNDS_START_COMMAND = new EditBoundsStartCommand();
```

Comment:
I wouldnt register the command unless you have a end command to go with it. Dont want to add a half done job

Reply:
ok XD